postinstall.js

```diff
@@ -7,6 +7,14 @@ const {checkSimpleGitHooksInDependencies, getProjectRootDirectoryFromNodeModules
  * Creates the pre-commit from command in config by default
  */
 async function postinstall() {
+
+    const {SKIP_INSTALL_SIMPLE_GIT_HOOKS} = process.env
```

Comment:
1. Maybe we should move this check into `simple-git-hooks.js` and export it for reusing?
2. There is no test case for it.